services/core/shared/core/compat/sdk_v5/transactions.js

```diff
@@ -234,6 +266,24 @@ const getTransactions = async params => {
 				transaction.username = account && account.username ? account.username : undefined;
 				transaction.isPending = false;
 
+				// For recipient info
+				if (transaction.asset.recipientAddress) {
+					const { recipientAddress, ...asset } = transaction.asset;
+					const recipientInfo = await getIndexedAccountInfo({
+						address: recipientAddress,
+					});
+					transaction.asset = asset;
+					transaction.asset.recipient = {};
+					transaction.asset.recipient = {
+						address: recipientInfo
+							&& (recipientInfo.address !== null) ? recipientInfo.address : undefined,
+						publicKey: recipientInfo
+							&& (recipientInfo.publicKey !== null) ? recipientInfo.publicKey : undefined,
+						username: recipientInfo
+							&& (recipientInfo.username !== null) ? recipientInfo.username : undefined,
```

Comment:
This can be reduced to:

```suggestion
						address: recipientInfo ? recipientInfo.address : undefined,
						publicKey: recipientInfo ? recipientInfo.publicKey : undefined,
						username: recipientInfo ? recipientInfo.username : undefined,
```

Reply:
Clarified offline!